doc/usecases/current/membership_application.md

```diff
@@ -0,0 +1,42 @@
+# Use Case: Membership Application
+
+## Summary
+
+- **Scope:** Membership
+- **Level:** User Goal
+- **Actors:** User (Membership Applicant), Administrator (Chairman)
+- **Brief:** The user can apply for a membership (account)
+- **Assignee:** Thorben (Frontend), Lukas (Keycloak Installation)
+
+## Scenarios
+
+- **Precondition:**
+  The user has opened the app and is not logged in.
+- **Main success scenario:**
+  1. The user can click an register button which leads them to Keycloak to fill out:
+     - username
+     - first+last name
+     - email address
+  2. The user can look at public maps (read-only and without seeing addresses!)
+  3. The user can visit a 3-step form to apply for membership, with:
+     - user photo (for Nextcloud)
+     - telephone number (optional)
+     - website
+     - organisation
+     - experience (optional)
+     - photo of map
+     - location of map (not shown to public users)
+     - bank account
+     - which kind of membership:
+       - paid subscription membership
```

Comment:
> Does a person that chooses the paid subscription permanently lose the ability to forgo paying for a year with gathering enough permacoins?

Only for the first year. The user can change the type of subscription every year. No reapplication is needed, it can be changed in the user settings.